### PR TITLE
Changes to measurements for data collection

### PR DESCRIPTION
**Pull request by reethikar**
After one round of internal data collection, we're adapting the measurements for a larger round of data collection:
- [x] Send more websocket RTT measurements, and have a cooldown period between websocket pings. 
- [x] Add another field to input form to get more info about user/VPN location. 
- [x] Use minimum found RTT for both websocket measurement and ICMP ping measurements instead of mean/median.

### CODE REVIEW

**Comment by reethikar** [comment]
@NullHypothesis I've made the changes we talked about in our last call, to get better data from #announcements testing tomorrow. I've also merged unit-tests into this branch so merging is easier (I made changes to unit-tests today to fix a bug)

File: handlers.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func measureHandler(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		}
-		details, err := validateForm(r.FormValue("email"), r.FormValue("exp_type"))
+		details, err := validateForm(r.FormValue("email"), r.FormValue("exp_type"), r.FormValue("location_vpn"), r.FormValue("location_user"))
```
Comment (NullHypothesis) [inline handlers.go:35]: It didn't occur to me to also ask for the user's location. That's a good idea!

File: measure.html [hunk 2]
```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
...
-        <input type="radio" id="exp_type" name="exp_type" value="direct">
+        <input type="radio" id="exp_type" name="exp_type" value="direct" onclick="document.getElementById('locinfo').style.display='none'">
         <label for="direct">Direct Connection</label><br>
-        <input type="radio" id="exp_type" name="exp_type" value="vpn">
+        <input type="radio" id="exp_type" name="exp_type" value="vpn" onclick="document.getElementById('locinfo').style.display='block'">
         <label for="vpn">VPN</label><br>
+        <div id="locinfo" style="display:none;">
+          <label>Location of VPN server connected (enter city, state):</label><br>
+          <input type="text" name="location_vpn"><br />
+          <label>Location of user (enter city, state):</label><br>
```
Comment (NullHypothesis) [inline measure.html]: Some rephrasing to make it clear that we're asking about the user's location.
```suggestion
          <label>Your location:</label><br>
```

File: measure.html [hunk 3]
```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
...
+          <input type="text" name="location_user"><br />
```
Comment (NullHypothesis) [inline measure.html]:
```suggestion
          <input type="text" name="location_user" placeholder="City, state (if applicable), country"><br />
```

File: measure.html [hunk 4]
```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
...
         <input type="text" placeholder="username@brave.com" name="email"><br />
         <label>Are you connecting to us through:</label><br>
-        <input type="radio" id="exp_type" name="exp_type" value="direct">
+        <input type="radio" id="exp_type" name="exp_type" value="direct" onclick="document.getElementById('locinfo').style.display='none'">
         <label for="direct">Direct Connection</label><br>
-        <input type="radio" id="exp_type" name="exp_type" value="vpn">
+        <input type="radio" id="exp_type" name="exp_type" value="vpn" onclick="document.getElementById('locinfo').style.display='block'">
         <label for="vpn">VPN</label><br>
+        <div id="locinfo" style="display:none;">
+          <label>Location of VPN server connected (enter city, state):</label><br>
```
Comment (NullHypothesis) [inline measure.html]:
```suggestion
          <label>Location of VPN server connected to (if any):</label><br>
```

File: measure.html [hunk 5]
```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
...
+          <input type="text" name="location_vpn"><br />
```
Comment (NullHypothesis) [inline measure.html]:
```suggestion
          <input type="text" name="location_vpn" placeholder="City, state (if applicable), country"><br />
```

File: measure.html [hunk 6]
```diff
@@ -38,10 +38,18 @@ <h1>Experiment Details</h1>
...
+          <label>Location of user (enter city, state):</label><br>
+          <input type="text" name="location_user"><br />
+          <label>We only use this information to reason about the measured latencies and physical distance.</label><br>
+          <label>Please enter <i>unknown</i> if you are unsure about VPN server location, or <i>"best available"</i> if you chose that on your VPN.</label><br>
```
Comment (NullHypothesis) [inline measure.html]:
```suggestion
          <label>Please enter <i>"unknown"</i> if you are unsure about your VPN server's location, or <i>"best available"</i> if you chose that in your VPN settings.</label><br>
```

**Review by NullHypothesis — COMMENTED** [review]
Left a bunch of suggestions to improve the clarity of the form.

File: measure.html (hunk 2 above)
Comment (reethikar) [reply measure.html]: Done in 0da43b5

File: measure.html (hunk 3 above)
Comment (reethikar) [reply measure.html]: Done and adjusted the size in 3cfb1e2

File: measure.html (hunk 5 above)
Comment (reethikar) [reply measure.html]: Done in 3cfb1e2

File: measure.html (hunk 6 above)
Comment (reethikar) [reply measure.html]: Done in b2455d3

**Pull request merged**